src/semgrep_agent/semgrep_app.py

```diff
@@ -74,6 +74,7 @@ def report_start(self, meta: GitMeta) -> None:
                 ignore_patterns=glom(body, T["scan"]["meta"].get("ignored_files", [])),
             )
             debug_echo(f"=== Our scan object is: {self.scan!r}")
+            return str(glom(body, T["policy"])) if "policy" in body else None
```

Comment:
```suggestion
            return glom(body, T["policy"], default=None)
```

Could you try if this is equivalent? I'm 90% sure yes.

Reply:
mypy gets upset if you don't cast the return as either a None or a string, and if you add str(...) around the whole thing here it will not return None properly. Other ideas? I could just make return type Any but I think that is just trading one form of clunkiness for another

Reply:
```suggestion
            from typing import cast, Optional
            return cast(Optional[str], glom(body, T["policy"], default=None))
```

I just found this mypy feature an hour ago, let's try it out? :D
Could you try if this is equivalent? I'm 90% sure yes.